README.md

```diff
@@ -1,2 +1,132 @@
-# final-capstone-frontend
-Final capstone frontend
+# EASY BOOKING APP
+
+The `EASY BOOKING` is a hotel booking app where users can make room reservations by specifying the date of entry and date for depature. Users will have various options to select from, based on what accomodities or special facilities offered along with a preferred room.
+
+## Screenshots
+
+<table>
+  <tr>
+    <tr>
+      <th>Splash</th>
+      <th>Home</th>
+    </tr>
+    <tr>
+      <td><img src="https://user-images.githubusercontent.com/101924220/199963390-fefe999e-7a19-4b15-9f77-5b38eff8c3a7.PNG
+" alt="Splash" align="center" /></td>
+      <td><img src="https://user-images.githubusercontent.com/101924220/199963396-da332516-a9a7-48cf-9580-dd4f4b1c210d.PNG
+" alt="Home" align="center" /></td>
+    </tr>
+  </tr>
+  
+  <tr>
+    <tr>
+      <th>Login</th>
+      <th>Register</th>
+    </tr>
+    <tr>
+      <td><img src="https://user-images.githubusercontent.com/101924220/199963379-be0480f7-8994-4297-ada0-bd7d82d22f3f.PNG
+" alt="Login" align="center" /></td>
+      <td><img src="https://user-images.githubusercontent.com/101924220/199963385-5417f0ed-c452-481f-8a73-897bbfc75802.PNG
+" alt="Register" align="center" /></td>
+    </tr>
+  </tr>
+</table>
```

Comment:
- [ ] These screenshots are not showing up:


![image](https://user-images.githubusercontent.com/57421823/200045754-06e87e96-7898-491b-9635-7a26b41695b7.png)

Kindly fix those in order to showcase your app 💪